pkg/util/config/config.go

```diff
@@ -66,12 +66,21 @@ type Scalyr struct {
 	ScalyrMemoryLimit   string `name:"scalyr_memory_limit" default:"1Gi"`
 }
 
+// LogicalBackup
+type LogicalBackup struct {
+	EnableLogicalBackup      bool   `name:"enable_logical_backup" default:"false"`
```

Comment:
Do we even need a separate flag for that? We can enable this functionality when any of `logical_backup_schedule`, `logical_backup_docker_image`, `logical_backup_s3_bucket` is not empty.

Reply:
makes sense, will update the docs as well